eng/Versions.props

```diff
@@ -8,7 +8,7 @@
     <MajorVersion>3</MajorVersion>
     <MinorVersion>8</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>4</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>5</PreReleaseVersionLabel>
```

Comment:
@JoeRobich 
Is this version change needed to flow to master?
What I am thinking it that master should remain 1 right?

Reply:
That is correct. Master should remain on prerelease 1.